src/ontology/mondo-edit.obo

```diff
@@ -619468,6 +619466,29 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4585 {source="PMID:27683935"} ! GRIN2A
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9063" xsd:anyURI
 
+[Term]
+id: MONDO:1060144
+name: retinitis pigmentosa 7, digenic
+def: "A digenic form of retinitis pigmentosa resulting from a mutation in the PRPH2 gene and a null mutation of the ROM1 gene, leading to progressive degeneration of the retina and vision loss." [OMIM:608133]
+xref: OMIM:608133 {source="MONDO:includedEntryInOMIM"}
+is_a: MONDO:0019200 {source="OMIM:608133"} ! retinitis pigmentosa
+is_a: MONDO:1040055 {source="OMIM:608133"} ! PRPH2-related retinopathy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10254 {source="OMIM:608133"} ! ROM1
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8424" xsd:anyURI
+
+[Term]
+id: MONDO:1060145
+name: Leber congenital amaurosis 18
+def: "A PRPH2-related retinopathy that causes retinal dystrophy of earlier onset." [OMIM:608133]
+synonym: "LCA18" EXACT ABBREVIATION [OMIM:608133]
+xref: OMIM:608133 {source="MONDO:includedEntryInOMIM"}
+is_a: MONDO:0018998 {source="OMIM:608133"} ! Leber congenital amaurosis
+is_a: MONDO:1040055 {source="OMIM:608133"} ! PRPH2-related retinopathy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9942 {source="OMIM:608133"} ! PRPH2
```

Comment:
same as above. This term should be added to the list of terms to not be updated via the omim automated update.